### PR TITLE
Add AirFrance-KLM Group and AirFrance Connect

### PR DESCRIPTION
**Pull request by seia-soto**
refs https://github.com/easylist/easylist/pull/19866

### CODE REVIEW

**Comment by seia-soto** [comment]
Including some privately shared information https://ghostery.slack.com/archives/C06B1NSDZPS/p1724338806062849

File: db/organizations/airfranceklm.eno [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name: Air France-KLM Group
+website_url: https://www.airfranceklm.com/en
+privacy_policy_url: https://wwws.airfrance.fr/en/information/legal/edito-psc
+privacy_contact:
+country:
+description: Air France-KLM Group provide passenger and cargo transportation services, including scheduled flights, freight forwarding, and logistical support across global routes.
```
Comment (philipp-classen) [inline db/organizations/airfranceklm.eno:6]:
```suggestion
description: Air France-KLM Group provides passenger and cargo transportation services, including scheduled flights, freight forwarding, and logistical support across global routes.
```

File: db/organizations/airfranceklm.eno [hunk 2]
```diff
@@ -0,0 +1,9 @@
+name: Air France-KLM Group
+website_url: https://www.airfranceklm.com/en
+privacy_policy_url: https://wwws.airfrance.fr/en/information/legal/edito-psc
+privacy_contact:
+country:
```
Comment (philipp-classen) [inline db/organizations/airfranceklm.eno:5]:
```suggestion
privacy_contact: mail.data.protection@airfrance.fr
country: FR
```

Comment (seia-soto) [reply db/organizations/airfranceklm.eno:5]: AirFrance and KLM are one company now (sharing Flying Blue program) but I'm not sure about the nationality. Would it be sufficient to set the country as FR?

Comment (philipp-classen) [reply db/organizations/airfranceklm.eno:5]: https://en.wikipedia.org/wiki/Air_France-KLM

Headquartered in Paris, France.

Comment (seia-soto) [reply db/organizations/airfranceklm.eno:5]: Makes sense!

**Review by philipp-classen — APPROVED** [review]

**Pull request merged**